configure.ac

```diff
@@ -39,6 +39,16 @@ AM_COND_IF([WITH_SQLITE3], [USE_SQLITE3="yes"], [USE_SQLITE3="no"])
 
 AC_SUBST([SQLITE3_LIBS])
 
+AC_ARG_ENABLE([broadcast],
+    AS_HELP_STRING([--enable-broadcast], [enable broadcast of events using local sockets (currently experimental)]))
+
+AS_IF([test "x$enable_broadcast" = "xyes" || test "x$enable_all" == "xyes"], [
+  AC_DEFINE(HAVE_BROADCAST,1,"broadcast RAS events")
+  AC_SUBST([WITH_BROADCAST])
+])
+
```

Comment:
The name of this feature ("broadcast") doesn't look nice, as this term is more used for TCP/IP special messages that are sent to a range of machines. 

The way this feature is mapped, it is actually adding an Unix socket for communication with other process(es).